modules/tracker/mbt/src/vpMbGenericTracker.cpp

```diff
@@ -484,6 +493,26 @@ void vpMbGenericTracker::computeVVS(std::map<std::string, const vpImage<unsigned
     iter++;
   }
 
+  // Update features number
+  for (std::map<std::string, TrackerWrapper *>::const_iterator it = m_mapOfTrackers.begin();
+       it != m_mapOfTrackers.end(); ++it) {
+    TrackerWrapper *tracker = it->second;
+    if (tracker->m_trackerType & EDGE_TRACKER) {
+      m_nb_feat_edge = tracker->m_error_edge.size();
```

Comment:
```suggestion
      m_nb_feat_edge += tracker->m_error_edge.size();
```

To handle multi-cameras case (e.g. stereo edge tracking) and following the description of `getNbFeaturesEdge()`

> Return the number of moving-edges features taken into account in the virtual visual-servoing scheme.